packages/core/src/dispatcher.ts

```diff
@@ -31,7 +31,8 @@ import { CAR, CARFactory, CarBlock } from 'cartonne'
 import all from 'it-all'
 
 const IPFS_GET_RETRIES = 3
-const DEFAULT_IPFS_GET_TIMEOUT = 30000 // 30 seconds per retry, 3 retries = 90 seconds total timeout
+const DEFAULT_IPFS_GET_SYNC_TIMEOUT = 30000 // 30 seconds per retry, 3 retries = 90 seconds total timeout
+const DEFAULT_IPFS_GET_LOCAL_TIMEOUT = 3000 // 3 seconds to get data from local ipfs store
```

Comment:
yeah, I was being conservative because I know ipfs gets overloaded and slows down under load.  Really this shouldn't take more than a few milliseconds, but I just don't really trust IPFS to actually perform that well.
Happy to drop it further though.  I originally considered 1 second.  We could even try 500 millis though that starts to make me nervous without testing the ipfs behavior here more.  WDYT?

CC @smrz2001

Reply:
changed to 1 second for now